Packs/GoogleMaps/Integrations/GoogleMaps/GoogleMaps.py

```diff
@@ -0,0 +1,126 @@
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+# Disable insecure warnings
+OUTPUTS_PREFIX = 'GoogleMaps'
+NO_RESULTS_MESSAGE = 'No matching places were found.'
+
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+class Client(BaseClient):
+    def __init__(self, api_key: str, base_url: str, proxy: bool, insecure: bool):
+        """
+        Client to use in the IPinfo integration. Uses BaseClient
+        """
+        super().__init__(base_url=base_url,
+                         verify=not insecure,
+                         proxy=proxy)
+        self.api_key = api_key
+
+    def google_maps_geocode(self, search_address: str, error_on_no_results: bool) -> List[CommandResults]:
+        # noinspection PyTypeChecker
+        response: Dict[str, Any] = self._http_request(method='GET', url_suffix='geocode/json?',
+                                                      params=assign_params(address=search_address, key=self.api_key))
+
+        status = demisto.get(response, 'status')
+        if status == 'OK':
+            return parse_response(response, search_address)
+
+        elif status == 'ZERO_RESULTS':
+            if error_on_no_results:
+                raise DemistoException(message=NO_RESULTS_MESSAGE, res=response)
+
+            return [CommandResults(readable_output=NO_RESULTS_MESSAGE)]
+
+        else:
+            error_message = demisto.get(response, 'error_message') or 'See raw response for details.'
+            raise DemistoException(message=error_message, res=response)
+
+
+def google_maps_geocode_command(client: Client, search_address: str, error_on_no_results: bool) -> List[CommandResults]:
+    return client.google_maps_geocode(search_address=search_address,
+                                      error_on_no_results=error_on_no_results)
+
+
+def test_module(client: Client) -> str:
+    """Tests GoogleMaps by geocoding the address of Demisto's (original) HQ"""
```

Comment:
No need to give the details of the easter egg :)
```suggestion
    """Tests GoogleMaps by geocoding a specific address"""
```